sunpy/instr/aia/__init__.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+"""
+Tools for prepping data from and calculating response
+functions for the Atmospheric Imaging Assembly (AIA)
+on the Solar Dynamics Observatory (SDO).
+"""
+from .aiaprep import aiaprep
+from .response import Response
```

Comment:
As the response file has the `h5py` dependency, I think it would be better if you had to import it separately from `instr.aia` so that you don't need `h5py` to import `aiaprep`.